BHoM_Adapter/AdapterActions/Move.cs

```diff
@@ -41,10 +41,22 @@ public abstract partial class BHoMAdapter
         /******************************************************/
         /* These methods represent Actions that the Adapter can complete. 
            They are publicly available in the UI as individual components, e.g. in Grasshopper, under BHoM/Adapters tab. */
+      
+        [Description("Performs a set up for the Request input of the Move Action.")]
+        public virtual bool VerifyMoveAdapters(BHoMAdapter source, BHoMAdapter target)
+        {
+            if (source == null || target == null)
+            {
+                Engine.Reflection.Compute.RecordError("Adapter input cannot be null.");
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
Do not think we need this method. This is not doing any setup, but simple null checks. To me, this belongs in the Move action.